test/client/src/block_builder.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+use crate::Client;
+use sc_block_builder::{BlockBuilder, BlockBuilderApi};
+use sc_client_api::backend;
+use sp_api::{ApiExt, BlockT, ProvideRuntimeApi};
+use sp_blockchain::Error;
+use sp_core::ExecutionContext;
+use sp_runtime::generic::BlockId;
+
+/// Push the inherents to a block so you don't have to.
+pub trait PushInherents {
```

Comment:
Also not true and for "one function" traits I often like to have the trait and the function the same name (while the function is in snake_case)

Reply:
Totally agree!